lib/intrinsicclass.ts

```diff
@@ -134,6 +134,3 @@ export function DefineIntrinsic<KeyT extends IntrinsicDefinitionKeys>(name: KeyT
   if (INTRINSICS[key] !== undefined) throw new Error(`intrinsic ${name} already exists`);
   INTRINSICS[key] = value;
 }
-export function GetIntrinsic<KeyT extends keyof typeof INTRINSICS>(intrinsic: KeyT): typeof INTRINSICS[KeyT] {
```

Comment:
Not sure this can be completely removed, even if the imports work, because there are some objects and methods (e.g. `from` methods, maybe intrinsic prototypes) that still need to be accessed.